arrow-site/doc/docs/arrow-core/datatypes/statet/README.md

```diff
@@ -211,10 +211,10 @@ fun main() {
 }
 ```
 
-As is immediately clear, this code while properly modelling the errors, has become more complex but our signature now represents a simple `Stack` as a `List` with an error domain.
+As is immediately clear, this code, while properly modelling the errors, has become more complex. But our signature now represents a simple `Stack` as a `List` with an error domain.
```

Comment:
Again, a sentence shouldn't start with a conjunction (but). It should be "complex, but"